in-adapter-api/src/main/java/gohigher/application/ApplicationQueryController.java

```diff
@@ -4,27 +4,39 @@
 
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 import gohigher.application.port.in.ApplicationQueryPort;
-import gohigher.application.port.in.CalenderApplicationRequest;
-import gohigher.application.port.in.CalenderApplicationResponse;
+import gohigher.application.port.in.CalendarApplicationRequest;
+import gohigher.application.port.in.CalendarApplicationResponse;
+import gohigher.application.port.in.DateApplicationRequest;
+import gohigher.application.port.in.DateApplicationResponse;
 import gohigher.auth.support.Login;
 import gohigher.controller.response.GohigherResponse;
 import lombok.RequiredArgsConstructor;
 
 @RestController
 @RequiredArgsConstructor
+@RequestMapping("/v1/applications")
 public class ApplicationQueryController implements ApplicationQueryControllerDocs {
 
 	private final ApplicationQueryPort applicationQueryPort;
 
-	@GetMapping("/v1/applications/calender")
-	public ResponseEntity<GohigherResponse<List<CalenderApplicationResponse>>> findByMonth(@Login Long userId,
+	@GetMapping("/calendar")
+	public ResponseEntity<GohigherResponse<List<CalendarApplicationResponse>>> findByMonth(@Login Long userId,
 		@RequestParam int year, @RequestParam int month) {
-		CalenderApplicationRequest request = new CalenderApplicationRequest(userId, year, month);
-		List<CalenderApplicationResponse> response = applicationQueryPort.findByMonth(request);
+		CalendarApplicationRequest request = new CalendarApplicationRequest(userId, year, month);
+		List<CalendarApplicationResponse> response = applicationQueryPort.findByMonth(request);
+		return ResponseEntity.ok(GohigherResponse.success(response));
+	}
+
+	@GetMapping("/processes/date")
+	public ResponseEntity<GohigherResponse<List<DateApplicationResponse>>> findByDate(@Login Long userId,
```

Comment:
월별 조회를 `v1/applications/month ~~`, 
일별 조회를 `v1/applications/date ~~`의 형식으로 가보는건 어떨까요??

Reply:
그리고 지금 생각이 든 것인데 제가 만든 월별 조회 api하고 동석님이 만든 api 모두 쿼리파라미터가 아니고 pathVariable로 가는 것도 좋지 않을까?하는 생각도 드네요!

`v1/applications/month/2023-09`, `v1/applications/date/2023-09-14` 처럼요..! (막상 적어보니 이상해보이기도 하네요)

위와 같은 생각이 든 이유는 일별 조회 쿼리의 현재 Api가 `v1/applications/processes/date?date = ~`와 같이 고정된 path의 마지막 keyword와 queryParam의 key가 동일하여 어색하다는 느낌이 들었기 떄문입니다!

Reply:
```suggestion
	}

	@GetMapping("/calendar")
	public ResponseEntity<GohigherResponse<List<DateApplicationResponse>>> findByDate(@Login Long userId,
```

date 가 반복되어 어색하다면 동일 URI 에 대해 파라미터 값의 여부로 분기쳐주면 어떨까요?

Reply:
저는 현재의 쿼리파라미터 방식이 좋은 것 같습니다~!

Reply:
파라미터 여부로 분기처리가 안되더라구요ㅜㅜ저도 파라미터 방식을 선호하는게 파라미터가 필터링(범위 등)의 관점이고 pathVariable은 어느 특정 리소스로 접근한다는 걸 명시하는 것으로 알고있습니다!

Reply:
그리고 저도 `v1/applications/processes/date?date = ~` 이 어색해서 이런 방식을 사용하게 되었습니다.
`v1/applications/processes/date?value = ~` 도 조금 어색하고...